src/data/project-list.json

```diff
@@ -222,5 +222,12 @@
     "name": "Dungeon-mode",
     "link": "https://github.com/dungeon-mode/game",
     "description": ""
+  },
+    {
+    "id": "vignette",
+    "img": "",
```

Comment:
We already have a press kit available so you can just grab that one. @JcdeA, can you check if the current press kit covers this?